packages/core/src/interpreter.ts

```diff
@@ -429,12 +429,16 @@ export class Interpreter<
     this.listeners.add(listener);
 
     // Send current state to listener
-    if (this.status === InterpreterStatus.Running) {
+    if (this.status !== InterpreterStatus.NotStarted) {
```

Comment:
This is a small change in behavior. I was afraid to emit this at all times because `this.initialState` might be impure